### DISCUSSION TITLE
Added index and cover page.

### DISCUSSION THREAD

**Discussion by HappyCerberus** [HappyCerberus/book-cpp-algorithms#2] [Announcements]
This is a small update adding an index and a cover page.

<hr /><em>This discussion was created from the release <a href='https://github.com/HappyCerberus/book-cpp-algorithms/releases/tag/v0.1.1'>Added index and cover page.</a>.</em>

**Comment by shaoethan** [comment]
Great to see progress

**Reply by HappyCerberus** [reply]
Thanks.

One major update is still coming, it will cover the C++20 ranges more in-depth and also go over all the views introduced in C++20 and coming in C++23.

After that, I will see whether there is interest in more.
